README.md

```diff
@@ -1,12 +1,16 @@
-<p align="center"><img src="https://flarum.org/img/logo.png"></p>
+<p align="center"><img src="https://discuss.flarum.org/assets/logo-2q2rcevl.png"></p>
 
 <p align="center">
-<a href="https://travis-ci.org/flarum/core"><img src="https://travis-ci.org/flarum/core.svg" alt="Build Status"></a>
+<a href="https://github.com/flarum/core/actions"><img src="https://github.com/flarum/core/workflows/JavaScript/badge.svg" alt="Git Actions"></a>
+<a href="https://github.com/flarum/core/actions"><img src="https://github.com/flarum/core/workflows/Lint/badge.svg" alt="Git Actions"></a>
+<a href="https://github.com/flarum/core/actions"><img src="https://github.com/flarum/core/workflows/Tests/badge.svg" alt="Git Actions"></a>
```

Comment:
Let's link to the actual workflow, and give it a more descriptive name. :grinning: 

```suggestion
<a href="https://github.com/flarum/core/actions?query=workflow%3ATests"><img src="https://github.com/flarum/core/workflows/Tests/badge.svg" alt="PHP Tests"></a>
```